server/rest/src/test/java/org/infinispan/rest/resources/AbstractRestResourceTest.java

```diff
@@ -49,7 +49,7 @@
 public class AbstractRestResourceTest extends MultipleCacheManagersTest {
    public static final String REALM = "ApplicationRealm";
    public static final Subject ADMIN_USER = TestingUtil.makeSubject("ADMIN", ScriptingManager.SCRIPT_MANAGER_ROLE, ProtobufMetadataManager.SCHEMA_MANAGER_ROLE);
-   public static final Subject USER = TestingUtil.makeSubject("USER");
+   public static final Subject USER = TestingUtil.makeSubject("USER", ProtobufMetadataManager.SCHEMA_MANAGER_ROLE, ScriptingManager.SCRIPT_MANAGER_ROLE);
```

Comment:
The idea is to have ADMIN have all permissions, but USER be a regular user, otherwise we have the same two subjects with different names

Reply:
I was experiencing issues with tests because of the lack of protostream. but I'm going to recheck this

Reply:
and the task execution

Reply:
The ADMIN user is used to bootstrap the server, fire-up the REST endopint, etc, and the USER is the one who should be used in the rest client to perform requests. Do regular users need those two roles to interact with the rest endpoint?

Reply:
I changed the test